caom2/caom2/plane.py

```diff
@@ -1059,7 +1059,8 @@ def __init__(self, bounds=None, dimension=None, resolving_power=None,
         self.sample_size = sample_size
         self.bandpass_name = bandpass_name
         self.energy_bands = energy_bands
-        self.em_band = em_band
+        if em_band is not None:
+            self.energy_bands.add(em_band)
```

Comment:
Can the calling code use `energy_bands` instead?
It looks to me that this if is required to conditionally set the `em_band` attribute to avoid the deprecation warning.